app/src/androidTest/java/com/github/sdp/mediato/ExploreFragmentTest.java

```diff
@@ -95,25 +109,24 @@ public static void cleanDatabase() {
     // Test whether the explore text is displayed and contains the correct text
     @Test
     public void testExploreFragmentTextView() {
-        ViewInteraction exploreText = onView(withId(R.id.text_explore));
         exploreText.check(matches(isDisplayed()));
         exploreText.check(matches(withText("Explore")));
-        UserDatabase.database.getReference().setValue(null);
-        CollectionsDatabase.database.getReference().setValue(null);
     }
 
-    /**
-     * @TODO Fix this test to work with the CI
+    // Test that the refresh button is displayed
+    @Test
+    public void testRefreshButtonIsDisplayed() {
+        refreshButton.check(matches(isDisplayed()));
+        refreshButton.perform(click());
+    }
+
     //Test that all the reviews from the not followed users are displayed
     @Test
     public void testItemCount() throws InterruptedException {
         Thread.sleep(5000);
         ViewInteraction outerRecyclerView = onView(withId(R.id.explore_posts));
-        //assertRecyclerViewItemCount(R.id.explore_posts, 2);
-        //outerRecyclerView.check(matches(hasItemCount(2)));
-        UserDatabase.database.getReference().setValue(null);
-        CollectionsDatabase.database.getReference().setValue(null);
-    }*/
+        assertRecyclerViewItemCount(R.id.explore_posts, 2);

```

Comment:
You could add a comment explaining where the 2 comes from here. Also, it would be better to define `ViewInteraction outerRecyclerView` at the beginning of the class, the same as the other view interactions. Since it's not a recycler view of recycler views here, I would also change the name to `exploreRecyclerView` or something.

Reply:
The outerRecyclerView value is actually not used so I just removed it there.